README.markdown

```diff
@@ -1,68 +1,104 @@
-# DITA Open Toolkit [![Build Status][1]](http://travis-ci.org/dita-ot/dita-ot) [![Slack][7]](http://slack.dita-ot.org/)
```

Comment:
👍 Not necessarily if there's a clear preference for another style. 

_(So far, the Markdown files in our various repos use a mix of inline and reference links, some of which are numbered, others use words for the link identifiers, and others use implicit link names.)_

Since named link identifiers mimic DITA keys, and can be called with the [shortcut reference link syntax](https://spec.commonmark.org/0.18/#shortcut-reference-link) @eerohele proposed, I'm fine standardizing on those if that's what we want.